### PR TITLE
Update requirements.txt for raspberry pi builds

### PR DESCRIPTION
**Pull request by VanDavv**
Currently, when trying to install requirements using `requirements.txt` file on Raspberry, it fails, since specified version is not available in Raspbian, example below:

```
pi@raspberrypi:~ $ pip3 install opencv-python==4.2.0.34
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting opencv-python==4.2.0.34
  Could not find a version that satisfies the requirement opencv-python==4.2.0.34 (from versions: 3.4.2.16, 3.4.2.17, 3.4.3.18, 3.4.4.19, 3.4.6.27, 3.4.7.28, 4.0.1.24, 4.1.0.25, 4.1.1.26)
No matching distribution found for opencv-python==4.2.0.34
```

We can fix this, by not specifying exact version of `opencv-python` but minimum required version - and in our case we rely on OpenCV version 4.
So simply changing `opencv-python==4.2.0.34` to `opencv-python>4` fixes the issue and always installs latest available version during installation, like below:

```
pi@raspberrypi:~ $ pip3 install opencv-python>4
pi@raspberrypi:~ $ pip3 freeze | grep opencv-python
opencv-python==4.1.1.26
```

Also, fixed requests version (and tested that it works on RPi)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 numpy==1.18.3
-opencv-python==4.2.0.34
-requests
\ No newline at end of file
+opencv-python>4
+requests==2.24.0
```
[the remark this answers is not in the document]
Comment (Luxonis-Brandon) [reply requirements.txt:3]: Yes, I'm told that standard practice is to always require and exact version, but I would like to make our standard practice to not require that as it's super-annoying as a user to have to chase down specific versions if it's not necessary...  I think we should just do an effort on our side to make sure our stuff continues to work w/ new versions as they come out.

Thoughts?

[the remark this answers is not in the document]
Comment (VanDavv) [reply requirements.txt:3]: From my perspective it's best to hardcode version, as it's not "use only this specific version" but "we've tested our code against this package version and it works". 

However, the core of this PR is to change `opencv-python` version specification, not fixing `requests`, so I can remove it

[the remark this answers is not in the document]
Comment (szabi-luxonis) [reply requirements.txt:3]: I agree, @VanDavv .
In this case we can you python environment markers, to specify exact version per platform.
[https://www.python.org/dev/peps/pep-0508/](url)

Something like this ? :
```
numpy~=1.18
opencv-python~=4.2.0; platform_machine != "armv7l"
opencv-python~=4.1.0; platform_machine == "armv7l"
requests~=2.24
```

Or if we go with ==

```
numpy==1.18.3
opencv-python==4.2.0.34; platform_machine != "armv7l"
opencv-python==4.1.1.26; platform_machine == "armv7l"
requests==2.24.0

```

Thoughts?

[the remark this answers is not in the document]
Comment (VanDavv) [reply requirements.txt:4]: Thanks @szabi-luxonis, changes added and tested, seems fine to go

**Pull request merged**